pom.xml

```diff
@@ -495,6 +495,33 @@
                     <lineEnding>LF</lineEnding>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>io.sundr</groupId>
+                <artifactId>sundr-maven-plugin</artifactId>
+                <version>0.200.0</version>
```

Comment:
Please extract this to a property, like all other dependency versions

Reply:
Ha, then that's obviously wrong, feel free to extract that too, but no pressure :)